project/.pre-commit-config.yaml

```diff
@@ -1,47 +1,32 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
+# see https://github.com/VOLTTRON/volttron-core/
 repos:
 - repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v4.0.1
+  rev: v4.1.0
   hooks:
-  - id: check-added-large-files
-  - id: check-toml
   - id: check-yaml
+  - id: check-json
+  - id: check-toml
+  - id: check-xml
+  - id: forbid-new-submodules
   - id: end-of-file-fixer
   - id: trailing-whitespace
-- repo: https://github.com/psf/black
-  rev: 22.3.0
-  hooks:
-  - id: black
-- repo: https://github.com/pre-commit/mirrors-mypy
-  rev: v0.910-1
-  hooks:
-  - id: mypy
```

Comment:
Let's keep the mypy hook. That is one of the features that we are promising in the README : 

https://github.com/VOLTTRON/copier-poetry-volttron-agent/blob/develop/README.md

Reply:
ref: https://github.com/pre-commit/mirrors-mypy